### ISSUE TITLE
Modules: GLMakie extension

### ISSUE THREAD

**Issue by utkinis** [PTsolvers/FastIce.jl#23]
Make an extension module for visualising the data with GLMakie

- [ ] `volumeslices` should use the PostProcessing module to gather data to rank 0 and plot
- [ ] `contour` should call isosurface extraction and MPI gather from the PostProcessing module

**Comment by luraess** [comment]
Ideally this should also work with CairoMakie for running on servers.

**Comment by utkinis** [comment]
> Ideally this should also work with CairoMakie for running on servers.

Good point, it's actually just a Makie extension, not GLMakie in particular =)
